### PR TITLE
Bookstore - Components

### PR DESCRIPTION
**Pull request by Bhekisile**
## Changes

- Setup the router and added two routes
- Created components
- Updated README file
- Created a list of books and edited a form

### CODE REVIEW

File: src/components/RenderBooks.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import '../styles/RenderBooks.css';
+
+function RenderBooks() {
```
Comment (gemmen29) [inline src/components/RenderBooks.js]: - [ ] Kindly create a list containing individual books as per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/project_init_project_wth_comps_v3_0.md#:~:text=Contains%20a%20list%20to%20renders%20individual%20books).

File: src/components/RenderBooks.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  return (
```
Comment (gemmen29) [inline src/components/RenderBooks.js]: - [ ] For rendering a single book, Kindly use the created component `Book.js` and pass the `title` and `author` as props to it.

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @Bhekisile, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Strangeal — APPROVED** [review]
Hi @Bhekisile, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 





## Highlights
- Added book list ✅ 
- Used props ✅ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**